src/frontend/pages/waiting-room.tsx

```diff
@@ -22,6 +24,10 @@ export default function WaitingRoom(): ReactElement {
     } else if (message.timeout) {
       router.push(`/thank-you`);
     } else if (message.gameId) {
+      toast.notify('All Players Have Joined, Starting New Game!', {
+        duration: 2000,
+        position: 'bottom-left',
+      });
```

Comment:
Question on whether or not we even need this, since the waiting room within the next second will route the player to the next screen

Reply:
I would agree that this is mostly unnecessary in practice since it moves pretty quickly to the next screen. Down to remove this one, thoughts? @zefwang

Reply:
Sure, I'm fine with that